pkg/commands/workload.go

```diff
@@ -58,8 +58,11 @@ import (
 const (
 	AnnotationReservedKey     = "annotations"
 	MavenOverwrittenNoticeMsg = "Maven configuration flags have overwritten values provided by \"--params-yaml\"."
+	SourceProxyDomain         = "svc.cluster.local"
 )
 
+var LocalImageRepo = fmt.Sprintf("%s.%s.%s", source.SourceProxyService, source.SourceProxyNamespace, SourceProxyDomain)
```

Comment:
*Suggestion:* Per latest plan, we could combine these 3 constants into 1 field. 

We had made it configurable in first version with an intention to support change in lsp endpoint for any reason. However, we moved away from that plan and will have same name/namespaces across releases.

Reply:
decided to move the variable to be part of the local registry transport, which contains the other constants. Decided to keep it as variable since the other vars that it's composed from are actually needed in other parts of the code.

Reply:
I also thought about adding a new method `GetLocalImageRepo()` which would return this variable and that way there's no way the caller function would override this variable, what do you think?